### DISCUSSION TITLE
Alignment stretched horizontally doesn't work on tooltip component!

### DISCUSSION THREAD

**Discussion by Den4ik1992** [Budibase/budibase#10011] [Help]
Hello everyone,

After inserting a tooltip to a text field, the "Align stretched horizontally" option no longer works for adjusting the size of the textbox. Manually entering the width of the textbox in pixels is still possible, but this may cause the website to lose its responsiveness.

Does anyone have any suggestions for a workaround or ideas on what I might be doing wrong? I would greatly appreciate any help or advice on this issue. Thank you in advance!

![2023-03-14-09-19-06](https://user-images.githubusercontent.com/82113641/224939114-2dbffe30-1874-4dda-aad4-e99985a687ed.gif)

EDIT//

I added own CSS code to the component:

overflow: visible;
max-width: none;
width: 100%;
display: flex;
justify-content: space-between; 

but it doesn't seem to be working. I'm not even sure what I'm doing ^^

**Comment by melohagan** [comment]
Hey @Den4ik1992 

When it comes to custom components, it may be best to raise an issue on the component repository: https://github.com/andz-bb/budibase-component-tooltip

With that said, there's no obligation for plugins to be maintained.
